linchpin/linchpin.constants

```diff
@@ -26,6 +26,7 @@ external_providers_path = %(default_config_path)s/linchpin-x
 use_rundb_for_actions = False
 distill_data = False
 distill_on_error = False
+use_actual_rcs = False
```

Comment:
This also needs a corresponding example and explanation in the https://github.com/CentOS-PaaS-SIG/linchpin/blob/develop/docs/source/examples/workspace/linchpin.conf